src/storage/hummock_sdk/src/key.rs

```diff
@@ -153,6 +151,53 @@ fn next_key_no_alloc(key: &[u8]) -> Option<(&[u8], u8)> {
     Some((&key[..pos], key[pos] + 1))
 }
 
+/// compute the next full key of the given full key
+///
+/// if the `user_key` has no successor key, the result will be a empty vec
+
+pub fn next_full_key(full_key: &[u8]) -> Vec<u8> {
+    let (user_key, epoch) = split_key_epoch(full_key);
+    let next_epoch = prev_key(epoch);
```

Comment:
use `prev_key` for `next_epoch` is a little comfused.

add some notes or use another function for getting  `next_epoch`